src/main/java/hu/oe/nik/szfmv/automatedcar/sensor/Camera.java

```diff
@@ -0,0 +1,192 @@
+package hu.oe.nik.szfmv.automatedcar.sensor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import hu.oe.nik.szfmv.automatedcar.bus.VirtualFunctionBus;
+import hu.oe.nik.szfmv.automatedcar.bus.packets.CameraPacket;
+import hu.oe.nik.szfmv.automatedcar.systemcomponents.SystemComponent;
+import hu.oe.nik.szfmv.environment.WorldObject;
+import hu.oe.nik.szfmv.environment.worldobjectclasses.RoadSign;
+
+public class Camera extends SystemComponent implements ISensor {
+
+    private static final int VISUAL_RANGE = 80;
+    private static final int HAROMSZAZHATVAN = 80;
```

Comment:
Ezt angolosítani kéne mert már szólt érte Gergő!

Reply:
refactor után már nem is kellett ez a field, szóval ez is megoldva